packages/components/src/components/ScalarSidebar/ScalarSidebarGroup.vue

```diff
@@ -25,9 +25,13 @@ import ScalarSidebarGroupToggle from './ScalarSidebarGroupToggle.vue'
 import ScalarSidebarIndent from './ScalarSidebarIndent.vue'
 import { type SidebarGroupLevel, useSidebarGroups } from './useSidebarGroups'
 
-const { is = 'ul' } = defineProps<ScalarSidebarItemProps>()
+const { is = 'ul' } = defineProps<
+  ScalarSidebarItemProps & { modelValue: boolean }
+>()
 
-const open = defineModel<boolean>({ default: false })
```

Comment:
I still think it's reasonable for the sidebar to have it's own internalized state but 🤷‍♀️